Bottom: 409029c6c4cb7348e8d33ea0427be567c42ad4cc
Top:    a0ea2e99c797ceb07edf0318af5ef673d354a77a
Author: Parth Shah <parth@linux.ibm.com>
Date:   2019-07-09 14:01:41 +0530

Refresh of sched-topology-introduce

---

diff --git a/include/linux/sched/topology.h b/include/linux/sched/topology.h
index cc3c4b621cab..e6b045f4a591 100644
--- a/include/linux/sched/topology.h
+++ b/include/linux/sched/topology.h
@@ -164,7 +164,7 @@ static inline struct cpumask *sched_domain_span(struct sched_domain *sd)
 }
 
 #ifdef CONFIG_SCHED_SMT
-static inliner struct cpumask *sched_domain_cores(struct sched_domain *sd)
+static inline struct cpumask *sched_domain_cores(struct sched_domain *sd)
 {
 	return to_cpumask(sd->shared->core_mask);
 }
